src/irmin-pack/ext.ml

```diff
@@ -187,12 +183,64 @@ module Maker (Config : Conf.S) = struct
                   Lwt.fail e
               | e -> Lwt.fail e)
 
-        (** Stores share instances in memory, one sync is enough. *)
-        let sync t = Contents.CA.sync (contents_t t)
+        (** For crash consistency and SWMR consistency, the order in which we
```

Comment:
What does SWMR stand for?

Reply:
I think this for `Single Writer Multiple Reader`.